src/functions/getLicenseText.ts

```diff
@@ -47,10 +47,10 @@ export const getLicenseText = async (
   }
 
   // TODO: read the file specified by "SEE LICENSE IN"
-  const licensePattern = `${depPath}/LICEN@(S|C)E?(.*)`;
+  const licensePattern = `${depPath}//LICEN@(S|C)E?(-MIT)?(.*)`;
 
   const res = glob.sync(licensePattern, { nocase: true });
-  if (res.length) {
+  if (res.length === 1) {
```

Comment:
If more than one license file is found, I think it is correct to let the user specify the license file. 
Thus, I fixed the condition.